assets/xml/objects/object_lodmoon.xml

```diff
@@ -1,12 +1,12 @@
 ﻿<Root>
     <File Name="object_lodmoon" Segment="6">
```

Comment:
Since it's not immediately obvious what this object is, could you put a comment with a description before the contents of the `File` tag? Actually I'm quite tempted to ask for that across the board while we've only got a few done.